encrypted-media-respec.html

```diff
@@ -2111,20 +2099,44 @@
                     Destroy these license(s) and/or key(s).
                   </p>
                 </li>
-                <li>
-                  <p>
-                    If this object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
-                  </p>
-                </li>
               </ol>
             </li>
-            <li><p>Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.</p></li>
-            <li><p>Run the <a def-id="update-expiration-algorithm"></a> algorithm on the <var>session</var>, providing <code>NaN</code>.</p></li>
-            <li><p>Let <var>promise</var> be the <a def-id="closed"></a> attribute of the <var>session</var>.</p></li>
-            <li><p>Resolve <var>promise</var>.</p></li>
+            <li><p>Run the <a def-id="session-closed-algorithm"></a> algorithm on the <var>session</var>.</p></li>
           </ol>
         </section>
       </section>
+      
+      <section id="session-closed">
+        <h4>Session Closed</h4>
+        <p>The Session Closed algorithm updates the <a>MediaKeySession</a> state after a session has been closed.</p>
+        </p>
+        <div class="note">
+          <p>
+            The CDM may close a session at any point, such as when the session is no longer needed or when system resources are lost.
+            In that case, this algorithm MUST be run.
+          </p>
+          <p>Keys in other sessions SHOULD be unaffected, even if they have overlapping key IDs.</p>
```

Comment:
Unrelated to this change, it seems this `SHOULD` should be `MUST`. Otherwise, there is a difference in observable behavior. At a minimum, non-overlapping key IDs should be unaffected, but I think the spec should expect correct and consistent behavior unless there is a good reason not to.